api/PerformancePaintTiming.json

```diff
@@ -32,6 +32,76 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "FirstContentfulPaint": {
```

Comment:
```suggestion
      "first-contentful-paint": {
```

Reply:
In actually did this originally, but then looked up Largest Contentful Paint and noticed it had gone with the API name instead of the performance observer entry name:

https://github.com/mdn/browser-compat-data/blob/313b8844c6d58da0c4d557ce0072896233657551/api/LargestContentfulPaint.json#L1-L3

Same with Event Timing:
https://github.com/mdn/browser-compat-data/blob/313b8844c6d58da0c4d557ce0072896233657551/api/PerformanceEventTiming.json#L3

And Event Timing:
https://github.com/mdn/browser-compat-data/blob/313b8844c6d58da0c4d557ce0072896233657551/api/PerformanceElementTiming.json#L3

And lots of others.

So I went for consistency with those.

This one is slightly different in that we have two APIs under one spec name so could be argued the previous ones are the spec names, rather than the individual APIs, but I still think it's better to be consistent with the others.

WDYT?

Reply:
I think of `first-contentful-paint` and` first-paint` as the sub features of the `PerformancePaintTiming` interface. It's also the names web developers interact with as opposed to `FirstContentfulPaint` and `FirstContentfulPaint` which look like WebIDL interfaces but aren't.

Reply:
OK. Will go with your suggestions then.